<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userManager</class>
 <widget class="QWidget" name="userManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>添 加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_batchDel">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>批量删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
     <column>
      <property name="text">
       <string>全选 </string>
      </property>
      <property name="textAlignment">
       <set>AlignRight|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>分发用户</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>相对路径</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>原目录结构</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>密码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>端口</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../radar/YTH/Server/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
